client/src/components/Authenticate.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Authenticate extends Component
+{
+    static propTypes = {
+        member: PropTypes.object,
```

Comment:
I think having a `member` as the parent of these props isn't required. The MFA app doesn't need to worry about the "member" context, just what MFA methods are available etc.